Chaincase.UI/ViewModels/CoinJoinViewModel.cs

```diff
@@ -58,6 +64,23 @@ public CoinJoinViewModel(ChaincaseWalletManager walletManager, Config config, IN
             _notificationManager = notificationManager;
             CoinList = selectCoinsViewModel;
 
+            CoinList.RootList
+                .Connect()
+                .ObserveOn(RxApp.MainThreadScheduler)
+                .Bind(out _coinViewModels)
+                .Subscribe();
+
+            _isRegistered = _coinViewModels
+                .ToObservableChangeSet()
+                .AutoRefresh(x => x.Status)
+                .ToCollection()
+                .Select(x => x.Any(coin => coin.Status == SmartCoinStatus.MixingInputRegistration))
+                .Throttle(TimeSpan.FromSeconds(2))
+                .ToProperty(this, x => x.IsRegistered, scheduler: RxApp.MainThreadScheduler);
+
+            this.WhenAnyValue(x => x.IsRegistered)
+                .Subscribe(_ => ScheduleConfirmNotification(RoundTimesout));
```

Comment:
This approach honestly makes me cringe because it makes so many assumptions. The core one is that MixingInputRegistration status won't have a valid change more often than every 2 seconds. Which actually _does_ work in practice but it sits wrong with me. I will still merge this because it fixes the problem it's out to fix and at least listens to an ObservableChangeSet that could be independent from a particular ViewModel.